www/dexpisvgNoaka.xslt

```diff
@@ -414,4 +354,55 @@
         <xsl:copy />
     </xsl:template>
 
+
+    <!-- Template for labels(only nozzles have labels in NOAKADEXPI) -->
+    <xsl:template match="Nozzle/Label">
+        <xsl:param name="height" />
+        <xsl:variable name="ID" select="../@ID" />
+    <xsl:variable
+            name="displayText"
+            select="following-sibling::GenericAttributes/GenericAttribute[@Name='ObjectDisplayNameAssignmentClass' or @Name='LineDescriptionAssignmentClass']/@Value" />
+    <xsl:if
+            test="$displayText">
+            <a id="{concat('https://assetid.equinor.com/plantx#', $ID)}" class="node">
+                <text>
+                    <xsl:attribute name="x">
+                        <xsl:value-of select="Position/Location/@X | Text/Position/Location/@X" />
+                    </xsl:attribute>
+                    <xsl:attribute name="y">
+                        <xsl:value-of
+                            select="$height - (Position/Location/@Y | Text/Position/Location/@Y)" />
```

Comment:
```suggestion
                            select="$height - (Text/Position/Location/@Y)" />
```
Som over, Position rett på Label ser ut til å bare brukes for andre elementer enn Nozzle